### PR TITLE
[Feature-6996][server & ui] Add dependent tips about success or failure

### PR DESCRIPTION
**Pull request by shangeyao**
#6996

### CODE REVIEW

**Comment by shangeyao** [comment]
#6996

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue [hunk 1]
```diff
@@ -566,6 +568,13 @@
...
               })
             }
+            if (list) {
+              list.forEach((dependent) => {
+                if (dependent.dependentResult) {
+                  this.setDependResult(JSON.parse(dependent.dependentResult))
+                }
+              })
+            }
           })
```
Comment (wangyizhi1) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue:578]: Thank you for your contribution, this problem may be caused by my previous refactoring.
BTW, if there are multiple DEPENDENT tasks in a workflow, problems may arise here. You can consider using 
``taskInstance`` computed in ``formModel.vue`` to obtain the ``dependentResult`` of the current instance.

**Review by caishunfeng — COMMENTED** [review]
I seems the main change is for UI,  @songjianet please review this pr, thx.

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue (hunk 1 above)
Comment (shangeyao) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue:578]: use $refs?

Comment (shangeyao) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue:578]: image
image
image
@Wangyizhi1

Comment (wangyizhi1) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue:578]: I made a mistake. Sorry, there's no problem here.

**Review by wangyizhi1 — APPROVED** [review]
LGTM

**Review by lenboo — APPROVED** [review]
+1

**Comment by caishunfeng** [comment]
@shangeyao please resolve the conflicts.

**Comment by shangeyao** [comment]
New PR was related to #7571

**Pull request closed without merging**